bears/c_languages/CPPLintBear.py

```diff
@@ -26,12 +30,20 @@ class CPPLintBear:
     def create_arguments(filename, file, config_file,
                          max_line_length: int=79,
                          cpplint_ignore: typed_list(str)=(),
-                         cpplint_include: typed_list(str)=()):
+                         cpplint_include: typed_list(str)=(),
+                         indent_size: int=2,
+                         use_spaces: bool=2):
         """
         :param max_line_length: Maximum number of characters for a line.
         :param cpplint_ignore:  List of checkers to ignore.
         :param cpplint_include: List of checkers to explicitly enable.
+        :param use_spaces: Bool value whether to use spaces or not.
```

Comment:
`` True if spaces are to be used instead`` i think that is what is used almost everywhere 
maybe you should use that for consistency... 
and also align the param docs ( the should start at the same column)